### ISSUE TITLE
[BUG] Freebox Delta Player ping not working (port 7000)

### ISSUE THREAD

**Issue by Matsuo3rd** [securechicken/homebridge-freebox-player-delta#4]
## Environment and Input

|Item|Your information|
|---|---|
|**Homebridge Version**|1.3.4|
|**Plugin version**|1.2.4|
|**FreeBox player version (model and FW)**|Freebox Player Delta fw 1.0.19|
|**iOS version**|14.6|

### Expected function and references
When switched ON (eg. using physical remote control), Freebox Player Delta shall be reported as ON as well in the iOS Home App. This is not the case: status is always OFF.

### Bug
Ping feature is not working. Even if Freebox Player Delta is ON.
See log trace:
`[2021-7-13 12:26:05] [Freebox Player] Power state check: undefined, {"address":"192.168.1.68","port":7000,"attempts":1,"avg":null,"results":[{"seq":0,"err":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"192.168.1.68","port":7000}}]}`

Bottom line: is ping on port 7000 still valid on Freebox Player Delta?
### Steps to Reproduce
1. Switch ON Freebox Player Delta
2. From terminal, issue command
`nc -zv 192.168.1.68 7000`
nc: connectx to 192.168.1.68 port 7000 (tcp) failed: Connection refused

### Ideas
1. Alternative use of http://$FREEBOX_PLAYER_IP:54243/device.xml ... but needs to activate option "Activer le serveur de rendu UPnP" from the Player. Not activated by default.
2. Alternative use of Freebox API (http://mafreebox.freebox.fr/api/v8/player/<PLAYER_ID>/api/v6/status/) ... but needs to be authenticated (more work).

**Comment by securechicken** [comment]
Dear @Matsuo3rd , thanks for reporting this.

I cannot reproduce with my setup (Freebox Player Delta, FW 1.0.19) and confirm a plugin bug.
Status is correctly reported (although a little delay is possible before correct status report, due to network latency and the "bridge" infrastructure + "polling" approach of HomeBridge and HomeKit), and TCP ping from other hosts than HomeBridge succeed (both by hostname and IP, IP only is shown as example) when the Player is up:
```
nc -v 192.168.[REDACTED] 7000
Connection to 192.168.[REDACTED] port 7000 [tcp/afs3-fileserver] succeeded!
```

- Are you sure that the tested IP is actually associated to the Player when you run your tests (i.e. wrong IP, dynamic IP in your LAN so it could have changed since you identified it, etc.)? 
  - You should be able to reach other TCP ports on the Player to confirm. A mere Web browsing to 80 and 8080 should bring a nginx 404 default page, while a browsing to 9080 would yield `status=ok`
  -  Regular ping to the Player IP should work as well (even when turned off).
- Isn't the Player service on 7000 up and alive? A Player reboot won't hurt, to get sure... Also, is this possible that a thing you did may have disabled the service running on TCP 7000?
- Are your HomeBridge and testing hosts actually allowed (network or OS filtering) and capable (network routing issues) to reach the TCP port 7000 on the IP you tested?
- Is your HomeBridge Web UI actually reporting the same (wrong) status than the one you see in your Apple devices?

Regards

**Comment by Matsuo3rd** [comment]
Dear @securechicken ,

Thanks for your swift answer and confirming ping feature is actually working. This drove my investigation and the outcome is that AirMedia feature must be turned ON for ping on port 7000 to operate. If AirMedia is disabled, then ping won't work.

So, one need to check ON the option "Activer AirMedia pour les appareils Apple" from the Freebox Player UI (bottom) > REGLAGES > APPLICATIONS > AirMedia Video > Activer AirMedia pour les appareils Apple.

Might worth a comment in the README. I cannot tell if this option is activated by default.

Thanks again!

Closing ticket.

**Comment by securechicken** [comment]
Thanks for this very useful information.
Fixed README and updated plugin to allow selection between AirPlay & UPnP services (I couldn't find any other that is actually shutdown when the Player is OFF/sleeping, except these).
